.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "_themes/legumeinfo-jekyll-theme"]
+	path = _themes/legumeinfo-jekyll-theme
+	url = https://github.com/legumeinfo/legumeinfo-jekyll-theme.git
```

Comment:
Before this goes into production it should be updated to track a commit associated with a specific tag in the theme repo:
```bash
cd _themes/legumeinfo-jekyll-theme
git checkout v1.0.0
cd ..
git add legumeinfo-jekyll-theme
git commit -m "Moved theme submodule to v1.0.0"
git push
```